modules/nf-core/tiara/tiara/main.nf

```diff
@@ -0,0 +1,56 @@
+process TIARA_TIARA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "conda-forge::tiara=1.0.3"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/tiara:1.0.3' :
+        'quay.io/biocontainers/tiara:1.0.3' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("classification_*.out.{txt,txt.gz}")      , emit: classifications
+    tuple val(meta), path("log_*.out.{txt,txt.gz}")                 , emit: log
+    tuple val(meta), path("*.{fasta,fasta.gz}")                     , emit: fasta, optional: true
+    path "versions.yml"                                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '1.0.3' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    """
+    tiara -i ${fasta} \
+        -o ${prefix}.out.txt \
+        --threads ${task.cpus} \
+        ${args}
+
+    ## Rename classification files
+    if [ -f ${prefix}.out.txt ]; then
+        mv ${prefix}.out.txt classification_${prefix}.out.txt
+    else
+        mv ${prefix}.out.txt.gz classification_${prefix}.out.txt.gz
+    fi
```

Comment:
Why are you renaming these? This seems like something that should be added with `task.ext.prefix`.

Reply:
For a given `-o` flag, Tiara outputs two .txt files: a contig->classification TSV called ${prefix}.out.txt, and a log file called log_${prefix}.out.txt. I wanted to put these into separate output channels, and path("*.txt") would capture both, so I thought it would be easiest just to rename it so that the two output channels can be specified easily. Open to better solutions!

Reply:
You could use the prefix in the output to use the full file name instead of using wildcards. (You'll need to make `prefix` a global variable by removing the `def` in front of it

Reply:
Fixed!